Singer.API/ClientApp/src/app/modules/legalguardians/shared/search-care-user-dialog/search-care-user-dialog.component.html

```diff
@@ -0,0 +1,33 @@
+<h2 mat-dialog-title>Zorggebruiker zoeken</h2>
```

Comment:
Could this not have been a wrapper around the `CareUserSearchComponent` in `SharedModule`?

Reply:
Maybe, will resolve in the "ultimate-folder-cleanup"